packages/fluent/src/components/ScaleTypeChoiceGroup/index.tsx

```diff
@@ -51,7 +51,7 @@ export const ScaleTypeChoiceGroup: FC<ScaleTypeChoiceGroupProps> = ({
 	return (
 		<ChoiceGroup
 			styles={style}
-			label={label}
+			label={label as string}
```

Comment:
maybe this prop should actually not be optional if it is not optional in the ChoiceGroup

Reply:
Fixed by making non-optional in ScaleTypeChoiceGroup